src/main/kotlin/com/epam/carapi/dto/color/ColorCreateDto.kt

```diff
@@ -0,0 +1,15 @@
+package com.epam.carapi.dto.color
+
+import com.epam.carapi.entity.Color
+import com.epam.carapi.entity.Vehicle
+
+data class ColorCreateDto(
```

Comment:
As we discussed today at our sync-up meeting. This structure is overloaded as we must use the appropriate entity for the appropriate operation. It's not a good way to develop dto objects. You should use one entity for all DML operations (SELECT, INSERT, UPDATE, DELETE).

I see here 2 ways how to validate the input for the above operations:
1. Use `Validator` class in apache common library at service layer. There can be `Validator` classes to validate the appropriate entities (DealerValidator, VehicleValidator, etc.)
2. Use `NOT NULL` field option in the table definition. In this case the validation process will be delegated to DB. This option will be more preferable and easier to implement. Also, you can use domain limitation for some field to define available values range for each field.